core/kernel/tee_ta_manager.c

```diff
@@ -642,7 +642,7 @@ TEE_Result tee_ta_invoke_command(TEE_ErrorOrigin *err,
 	}
 
 	tee_ta_clear_busy(sess->ctx);
-	if (res != TEE_SUCCESS)
+	if (res != TEE_SUCCESS && res != TEE_ERROR_SHORT_BUFFER)
```

Comment:
It would be nice with a small comment on the thing with `TEE_ERROR_SHORT_BUFFER`